scanapi/__init__.py

```diff
@@ -53,12 +53,12 @@ def scan(spec_path, output_path, config_path, reporter, template, log_level):
     try:
         api_spec = load_config_file(spec_path)
     except FileNotFoundError as e:
-        error_message = f"Could not find API spec file: {spec_path}. {str(e)}"
-        logger.error(error_message)
+        error_msg = f"Could not find API spec file: {spec_path}. {str(e)}"
```

Comment:
I did this change only to test black lint :D

Reply:
was this supposed to be caught?

Reply:
It is not a lint error, but the file should be checked

Reply:
Let me add an error